float32 increment
---
bool success
string message
